go/bind/notifications.go

```diff
@@ -47,6 +48,19 @@ type ChatNotification struct {
 	BadgeCount          int
 }
 
+func HandlePostTextReply(strConvID, tlfName string, body string, replyTo int) (err error) {
+	outboxID, err := storage.NewOutboxID()
+	if err != nil {
+		return err
+	}
+	convID, err := chat1.MakeConvID(strConvID)
+	if err != nil {
+		return err
+	}
+	kbCtx.ChatHelper.SendTextByIDNonblock(context.Background(), convID, tlfName, body, &outboxID, nil)
```

Comment:
don't you need `&chat1.MessageID(replyTo)` here?

Reply:
That's actually old code, I originally did a reply to, but realized that isn't how anyone else does it and it felt weird. So I just post the message without making it a reply

Reply:
oh why not drop the `replyTo` param then?